models/CommentManager.php

```diff
@@ -0,0 +1,132 @@
+<?php
+// Permet d'avoir accès à la base de données
+require_once "Model.php";
+
+class CommentManager extends Model{
+    
+    //FONCTIONS POUR LE FRONTEND
+    // Fonction qui permet de récupérer les commentaires visibles d'un article (seen = 1)
+     function get_comments_blog(){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [get_comments_blog accesses the super-global variable $_GET.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)